validator/validator-main.protoascii

```diff
@@ -1217,7 +1217,7 @@ tags: {  # Special custom 'author' spreadsheet for AMP
       allowed_declarations: "voice-family"
       allowed_declarations: "width"
       allowed_declarations: "word-spacing"
-      allowed_declarations: "writing-mode"
+      allowed_declarations: "writing-mode"      
```

Comment:
why the trailing whitespace?

Reply:
reverted. Not intentional. It was from copy and paste. I initially deleted these as this whitelist logic will now be handled by the viewer but then saw that Greg made this change internally..which propagate to github eventually. Thanks